x/evm/types/genesis.go

```diff
@@ -29,7 +54,7 @@ type (
 	GenesisAccount struct {
 		Address ethcmn.Address   `json:"address"`
 		Balance *big.Int         `json:"balance"`
-		Code    []byte           `json:"code,omitempty"`
+		Code    Code             `json:"code,omitempty"`
```

Comment:
maybe we should use `ethcmn.Hex`? that would avoid us duplicating code

Reply:
ah good move, will update